conans/client/conf/detect.py

```diff
@@ -204,10 +203,13 @@ def _detect_gcc_libcxx(executable, version, output, profile_name, profile_path):
         error, output = detect_runner("%s main.cpp -std=c++11" % executable)
         if error:
             if "using libstdc++" in output:
+                output.info("gcc C++ standard library: libstdc++")
```

Comment:
I have to use another variable here, output is overridden at 203, now it's a string, not a output object.

Reply:
Good catch, fixed.